kubectl-cloudflow/deploy/deploy.go

```diff
@@ -92,6 +95,311 @@ func GetCloudflowApplicationDescriptorFromDockerImage(dockerRegistryURL string,
 	return spec, *pulledImage
 }
 
+// GetCloudflowApplicationSpecFromDockerImages pulls a image and extracts the Cloudflow Application descriptor from a docker label
+func GetCloudflowApplicationSpecFromDockerImages(imageRefs []ImageReference,
```

Comment:
the name can be improved, you don't really get the appspec from docker images. `CreateApplicationSpecFromBlueprintAndImages`.. ehhh ugh... can we maybe do this in two steps?
`GetStreamletDescriptors(imageRefs)`
`CreateCloudflowApplication(blueprint, streamletdescriptors)`
Something like that.

Reply:
ok .. will refactor

Reply:
refactored